microsoft-edge/webview2/how-to/webdriver.md

```diff
@@ -84,46 +84,35 @@ At this point you should have WebView2 Runtime installed, built a WebView2 proje
     using System.Threading.Tasks;
     ```  
     
-## Step 4: Drive WebView2 with Selenium and Microsoft Edge Driver  
+## Step 7: Drive WebView2 with Selenium and Microsoft Edge Driver  
```

Comment:
@jasonstephen15 - should I change "Step 7" back to "Step 4"?  (in my working branch for the Edit pass)  Or add new headings defining step 4, 5, & 6?